tox.ini

```diff
@@ -1,22 +1,33 @@
 [tox]
-minversion = 3.7.0
+minversion = 3.8.4
 envlist =
     lint
     py{27,35,36,37}-ansible{25,26,27}-{functional,unit}
     format-check
     doc
 skipdist = True
 skip_missing_interpreters = True
+isolated_build = True
 
 [testenv]
 usedevelop = True
 passenv = *
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
```

Comment:
Is using a `test` entry in `[options.extras_require]` an option in replacement for this? Then we can rely on a single source file for the package listing. I've been doing this a little in my recent experiments and it works quite nicely ... it could be an abuse of the `extra_requires` though ...

Reply:
Yes, I usually do this as well. Calling it `testing` though: https://github.com/cherrypy/cheroot/blob/92490dd/setup.cfg#L83-L104.

There's a downside to this, though. You cannot install extras only without installing a dist itself. It's one thing I've been discussing with one of the tox maintainers in October... It seems like this should be implemented on the pip/setuptools side.

Reply:
I'd prefer to postpone this change and do in in a separate tox refactoring PR.